### PR TITLE
Updating encoding samples

### PR DESCRIPTION
**Pull request by jonpayne**
## Purpose
Updating encoding samples to use the new Media Services SDK for .NET.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: VideoEncoding/Encoding_H264/Program.cs [hunk 1]
```diff
@@ -1,721 +1,523 @@
...
-                                    layers:  new H264Layer[]
+                                Channels = 2,
+                                SamplingRate = 48000,
+                                Bitrate = 128000,
+                                Profile = AacAudioProfile.AacLc
+                            },
+                            // Next, add a H.264 for the video encoding
+                            new H264Video
+                            {
+                                // Set the GOP interval to 2 seconds for all H.265 layers
```
Comment (xpouyat) [inline VideoEncoding/Encoding_H264/Program.cs:144]: H.265 is referenced but it's H.264 here

File: VideoEncoding/Encoding_H264_ContentAware_Constrained/README.md [hunk 2]
```diff
@@ -72,115 +72,24 @@ To learn more about Content Aware Encoding, see the article on [Content Aware En
 1. Creates an input asset and upload a media file into it.
 1. Submits a job and monitoring the job using polling method.
 1. Downloads the output asset.
-1. prints urls for streaming.
-
-> [!TIP]
-> The `Program.cs` file has extensive comments.
+1. Prints URLs for streaming.
```
Comment (xpouyat) [inline VideoEncoding/Encoding_H264_ContentAware_Constrained/README.md:75]: The C# code provided in the readme file must be updated. The class name changed from PresetConfigurations( to EncoderPresetConfigurations {} and parameters have now uppercase etc....

File: VideoEncoding/Encoding_HEVC_ContentAware/Program.cs [hunk 3]
```diff
@@ -1,650 +1,529 @@
...
+                            // Add an AAC Audio layer for the audio encoding
+                            new AacAudio
+                            {
+                                Channels = 2,
+                                SamplingRate = 48000,
+                                Bitrate = 128000,
+                                Profile = AacAudioProfile.AacLc
+                            },
+                            // Next, add a HEVC (H.265) for the video encoding
+                            new H265Video
```
Comment (xpouyat) [inline VideoEncoding/Encoding_HEVC_ContentAware/Program.cs:142]: This code does not setup Content Aware Encoding for HEVC., but a custom preset...

File: VideoEncoding/Encoding_OverlayImage/Program.cs [hunk 4]
```diff
@@ -1,643 +1,524 @@
...
-                            }
-                        },
-                        Codecs = new List<Codec>
-                        {
-                            new AacAudio
-                            {
-                            },
+                            // Add an AAC Audio layer for the audio encoding
+                            new AacAudio(),
+                            // Next, add a HEVC (H.265) for the video encoding
```
Comment (xpouyat) [inline VideoEncoding/Encoding_OverlayImage/Program.cs:154]: H.264 instead

File: VideoEncoding/Encoding_StitchTwoAssets/README.md [hunk 5]
```diff
@@ -4,99 +4,26 @@ This sample shows how to create a custom encoding Transform using the StandardEn
 1. Creates an input asset and upload a media file into it.
 1. Submits a job and monitoring the job using polling method.
 1. Downloads the output asset.
-1. prints urls for streaming.
-
-> [!TIP]
-> The `Program.cs` file has extensive comments.
+1. Prints URLs for streaming.
```
Comment (xpouyat) [inline VideoEncoding/Encoding_StitchTwoAssets/README.md:7]: Readme should be updated :this is a "stitching two assets" sample

**Review by xpouyat — CHANGES_REQUESTED** [review]

File: VideoEncoding/Encoding_H264/Program.cs (hunk 1 above)
Comment (jonpayne) [reply VideoEncoding/Encoding_H264/Program.cs]: I've updated the code to resolve this issue.

File: VideoEncoding/Encoding_H264_ContentAware_Constrained/README.md (hunk 2 above)
Comment (jonpayne) [reply VideoEncoding/Encoding_H264_ContentAware_Constrained/README.md:82]: I've updated the code to resolve this issue.

File: VideoEncoding/Encoding_OverlayImage/Program.cs (hunk 4 above)
Comment (jonpayne) [reply VideoEncoding/Encoding_OverlayImage/Program.cs]: I've updated the code to resolve this issue.

File: VideoEncoding/Encoding_HEVC_ContentAware/Program.cs (hunk 3 above)
Comment (jonpayne) [reply VideoEncoding/Encoding_HEVC_ContentAware/Program.cs]: I've updated the code to resolve this issue.

**Pull request merged**